pkgs/by-name/lo/logseq/package.nix

```diff
@@ -4,7 +4,7 @@
 , appimageTools
 , makeWrapper
 # graphs will not sync without matching upstream's major electron version
```

Comment:
I guess from the comment, `electron` should be `electron_30` here?

Reply:
Electron 28?

https://github.com/logseq/logseq/blob/master/package.json#L116

Reply:
Oh, yes then.